recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge libspatialite_dev
```

Comment:
This will go to a special tag `libspatialite_dev` so folks don't get it by mistake.